app/design/frontend/Diana/default-theme/Magento_Theme/layout/default.xml

```diff
@@ -9,9 +9,13 @@
     <body>
         <referenceContainer name="content">
             <block name="first.block.via.theme.default" template="Magento_Theme::first-template-via-theme-default.phtml" />
+            <block name="newsletter.subscribe.popup" template="Magento_Theme::newsletter-subscribe-popup.phtml" />
```

Comment:
@Dianaeva Newsletter subscribe will not work due to the absence of backend functionality. It should be fixed